.github/workflows/ci.yml

```diff
@@ -145,16 +145,16 @@ jobs:
         id: cache-api-code
         with:
           path: |
-            src/ansys/fluent/core/datamodel_231
-            src/ansys/fluent/core/fluent_version_231.py
-            src/ansys/fluent/core/meshing/tui_231.py
-            src/ansys/fluent/core/solver/settings_231
-            src/ansys/fluent/core/solver/tui_231.py
+            src/ansys/fluent/core/generated/datamodel_231
```

Comment:
Can we factor out these duplicated paths?

Reply:
Currently we are maintaining separate caches for each Fluent versions which are updated at different frequency, so not changing that ATM.